.github/workflows/test-macOS.yml

```diff
@@ -56,12 +56,3 @@ jobs:
       - name: Test
         if: ${{steps.skip-ci.outputs.RESULT != 'true'}}
         run: pnpm run test:jest
-
-      - name: Collect coverage
```

Comment:
This PR is intended to address the error noted in https://github.com/web-infra-dev/modern.js/pull/4795#issuecomment-1764056667. We can proceed with this task subsequently.

Reply:
Upload coverage report sometimes fails, usually we will try again to solve the problem.

We can delete `codecov-action` first because it has been discussed in https://github.com/web-infra-dev/modern.js/issues/4780